.github/workflows/CD.yml

```
@@ -0,0 +1,91 @@
+permissions:
+  contents: read
+
+jobs:
+  CD:
+    runs-on: ubuntu-latest
+    steps:
+    - name: 1. checkout
+      uses: actions/checkout@v4
+
+    - name: 2. Set up JDK 21
+      uses: actions/setup-java@v3
```

actionlint (1.7.4)

the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)